apps/ledger-live-desktop/src/renderer/screens/nft/Send/Summary.tsx

```diff
@@ -8,20 +8,40 @@ import { Transaction } from "@ledgerhq/live-common/generated/types";
 import Media from "~/renderer/components/Nft/Media";
 import Skeleton from "~/renderer/components/Nft/Skeleton";
 import { useNftMetadata } from "@ledgerhq/live-common/nft/NftMetadataProvider/index";
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
 import { centerEllipsis } from "~/renderer/styles/helpers";
-import { NFTMetadata } from "@ledgerhq/types-live";
+import { NFTMetadata, ProtoNFT } from "@ledgerhq/types-live";
+import { getNFT } from "@ledgerhq/live-common/nft/index";
+
 type Props = {
   transaction: Transaction;
+  currency: CryptoCurrency;
 };
 const Summary = ({ transaction }: Props) => {
-  const allNfts = useSelector(getAllNFTs);
-  // @ts-expect-error error from type in families
-  const [tokenId] = transaction.tokenIds;
-  // @ts-expect-error error from type in families
-  const [quantity] = transaction.quantities;
-  // @ts-expect-error error from type in families
-  const contract = transaction.collection;
-  const nft = allNfts.find(nft => nft?.tokenId === tokenId && nft?.contract === contract);
+  const allNfts = useSelector(getAllNFTs) as ProtoNFT[];
+  const tokenId = useMemo(() => {
+    if (transaction.family === "evm") {
+      return transaction.nft?.tokenId;
+    } else if (transaction.family === "ethereum") {
+      return transaction.tokenIds?.[0];
+    }
```

Comment:
i agree we need to avoid doing these family conditions, especially now that we have a clear type on what a Coin family exposes (see families/types.ts)

family can simply expose a way to infer these from the transaction

like 

```js
const specific = getLLDCoinFamily(transaction.family)
specific?.getTokenId(transaction)
```

possibly can be factorize into a single function that will get you `{tokenId,quantity,contract}`

Reply:
@gre did you read my answer on slack? These if are only during the eth > evm migration, before that the whole NFT implem has been standard for more than 1y now.
Modularizing the NFT feature it could be considered, but out of scope for this PR.

Reply:
if we go with this, let's at least track it with a // FIXME and a ticket 🙏

Reply:
I'll rework the LLD part with the `getLLDCoinFamily` as much as I can 👌